be/src/block_cache/cache_options.h

```diff
@@ -27,26 +28,30 @@ struct DirSpace {
 
 struct CacheOptions {
     // basic
-    size_t mem_space_size;
+    size_t mem_space_size = 0;
     std::vector<DirSpace> disk_spaces;
     std::string meta_path;
 
     // advanced
-    size_t block_size;
-    bool enable_checksum;
-    bool enable_direct_io;
+    size_t block_size = 0;
+    bool enable_checksum = false;
+    bool enable_direct_io = false;
+    bool enable_through_data_mode = false;
     std::string engine;
-    size_t max_concurrent_inserts;
-    size_t max_flying_memory_mb;
-    bool enable_cache_adaptor;
-    size_t skip_read_factor;
+    size_t max_concurrent_inserts = 0;
+    size_t max_flying_memory_mb = 0;
+    double scheduler_threads_per_cpu = 0;
+    double skip_read_factor = 0;
 };
 
 struct WriteCacheOptions {
     // If ttl_seconds=0 (default), no ttl restriction will be set. If an old one exists, remove it.
     uint64_t ttl_seconds = 0;
     // If overwrite=true, the cache value will be replaced if it already exists.
-    bool overwrite = true;
+    bool overwrite = false;
```

Comment:
why change to false

Reply:
> why change to false

Because it can avoid write the same cache item repeatedly. In our case, updating an exist cache will not happen. If the remote file is changed, the cache key will be changed too, so we will cache it as a new cache item.